django/contrib/admin/static/admin/js/nav_sidebar.js

```diff
@@ -0,0 +1,22 @@
+let main = document.getElementById('main');
```

Comment:
This might just be personal taste but I find IIFEs to be quite ugly and a little hacky. If we don't want to pollute the global scope, and now that we have better language features available, I think a better solution would be to use `let` and put it in a block scope.

However, our ESLint setup doesn't seem to want us using ES6, although since https://github.com/django/django/pull/12502 we should be able to use it. I don't see the eslint config in the repo (but I didn't search everywhere). I assume it's in Jenkins? Could this be changed now? For such a small change I will probably just relent and use `var` and the IIFE though.

Reply:
Nevermind, the config was in an extremely obvious place and somehow I still missed it.

I've changed the eslint config to allow ES6 and have gone with my solution here. However, if you think it's better as an IIFE I will change it.

Reply:
Using the ES6 blocks is good idea. You may be interested in PR #12767 which aims to do this throughout the existing code.

Reply:
It's probably a good idea to merge #12767 first, as then I'll need to make a small change to the JS here.

Reply:
Grrr. Better look at #12767 then...  😀

Reply:
#12767 is in, so you can update here.

Reply:
Thanks Carlton, I've sorted this out now.